lighthouse-core/audits/diagnostics.js

```diff
@@ -67,7 +67,11 @@ class Diagnostics extends Audit {
     return {
       score: 1,
       rawValue: 1,
-      details: {items: [diagnostics]},
+      details: {
+        type: 'diagnostic',
+        // TODO: Consider not nesting diagnostics under `items`.
+        items: [diagnostics],
```

Comment:
awww this is just to remain non-breaking? :)

the idea would be `details: {type: 'diagnostic', ...diagnostics}` if we were writing it today right?

Reply:
> awww this is just to remain non-breaking?

Ha, well, since we just added this and we consider it internal only, I'm happy to change, I just didn't want to assume and/or cause churn in people's lives, especially because i mostly don't use these :)

We could change these newer ones and leave the older ones like `metrics` until v5 since we know of at least a few people who rely on them.

Reply:
I meant "awwww" as in how thoughtful :D I'm probably the only person that relies on this and that would indeed be a pain, lol

I like leaving them until v5 :)

Reply:
haha, got it. SGTM :)